tests/helpers.py

```diff
@@ -185,13 +186,31 @@ def start_controller(self, clean_config=False, enable_all=False, del_flows=False
             daemon += ' -E'
         os.system(daemon)
 
+        self.wait_controller_start()
+
+    def wait_controller_start(self):
+        """Wait until controller starts according to core/status API."""
+        wait_count = 0
+        while wait_count < 60:
+            try:
+                response = requests.get('http://127.0.0.1:8181/api/kytos/core/status/', timeout=1)
+                assert response.json()['response'] == 'running'
+                break
+            except:
```

Comment:
```suggestion
            except AssertionError:
```

Reply:
@ajoaoff thanks for the suggestion! I didn't use a specific exception in a first-place because the `request.get()` can also fail to provide a response (returning a Timeout or something like this), and the JSON parser could also fail (due to an incomplete response from the server). Then I decided to keep the generic exception handler. What do you think?